docs/elements.md

```diff
@@ -346,6 +347,59 @@ Attribute | Type | Default | Description
 
 -----
 
+## `pl-drag-drop-blocks` element
+
+Drag and drop to order answer tiles to form a (potentially ordered) list of answers. 
+
+#### Sample Element
+
+![](elements/pl-drag-drop-blocks.png)
+
+**question.html**
+```html
+<p>
+Drag all the prime numbers below to the yellow box. Note that the ordering of the answer does not matter, any permutation of prime numbers are accepted. Remember 1 is not a prime number.
+</p>
+<drag-drop-elements answers-name="prime-numbers-unordered" shuffle-options="true" permutation-mode="any">
+  <pl-answer correct="false">1</pl-answer>
+  <pl-answer correct="true">2</pl-answer>
+  <pl-answer correct="true">3</pl-answer>
+  <pl-answer correct="false">4</pl-answer>
+  <pl-answer correct="true">5</pl-answer>
+  <pl-answer correct="false">6</pl-answer>
+  <pl-answer correct="true">7</pl-answer>
+</drag-drop-elements>
+```
+#### Customizations
+
+Attribute | Type | Default | Description
+--- | --- | --- | ---
+`answers-name` | string | — | Variable name to store data in.
+`shuffle-options` | boolean | — | Specify whether the ordering of options on the left-hand-side should be shuffled. If false, the ordering in the HTML file will be used.
+`permutation-mode` | string | - | One of the following: `any`, `html-order`, `ranking`. See details below for description.
+
+Within the `pl-drag-drop-blocks` element, each answer tile must be specified with
```

Comment:
There is some inconsistent naming throughout. Is it going to be called "pl-drag-drop"? Or "pl-drag-drop-blocks"?

Reply:
the element is gonna use the existing name of the pl-parson element, I just used a different name as placeholder